dist/common/systemd/scylla-server.service

```diff
@@ -14,7 +14,7 @@ LimitNPROC=8096
 EnvironmentFile=/etc/sysconfig/scylla-server
 EnvironmentFile=/etc/scylla.d/*.conf
 ExecStartPre=/opt/scylladb/scripts/scylla_prepare
-ExecStart=/usr/bin/scylla $SCYLLA_ARGS $SEASTAR_IO $DEV_MODE $CPUSET
+ExecStart=/usr/bin/scylla $SCYLLA_ARGS $SEASTAR_IO $DEV_MODE $CPUSET $MEM_CONF
 ExecStopPost=/opt/scylladb/scripts/scylla_stop
 TimeoutSec=900
```

Comment:
Don't we also have to add CAP_IPC_LOCK?

Note, we put this now in capabilities.conf, but maybe we can move it back to scylla-server.service. systemd-219-33 was released more than 3 years ago, we can add a conflict with earlier versions.